index.rst

```diff
@@ -3,11 +3,6 @@
 Symfony Documentation
 =====================
 
-.. toctree::
-    :hidden:
-
-    changelog
```

Comment:
Does this need to be indexed? The spell checker checks all the documents, including hidden ones. I have "excluded" it instead above so that no warnings are shown.